code/api/tests/subscription.test.js

```diff
@@ -0,0 +1,29 @@
+import request from 'supertest';
+import express from 'express';
+import graphqlHTTP from 'express-graphql';
+import schema from '../src/setup/schema';
+
+describe("user queries", () => {
+  let server;
+
+  beforeAll(async () => {
+    server = express();
+
+    server.use(
+      '/',
+      graphqlHTTP(request => ({
+        schema: schema,
+        graphiql: false
+      })
+    ))
+  });
+
+  it('returns all subscriptions', async () => {
```

Comment:
With the seeds on master we'll be able to test this a little better, besides the auth queries